server/routes/index.js

```diff
@@ -0,0 +1,5 @@
+const participantUserRoute = require('./participant-user');
+
+module.exports = {
+  participantUserRoute,
+};
```

Comment:
This seems like a first step towards a larger scale refactor of how we use routes, yea?

Reply:
👍